src/extensions/addrbar/bootstrap-table-addrbar.js

```diff
@@ -63,6 +65,23 @@ function _buildUrl (dict, url = window.location.search) {
   return url
 }
 
+/*
+* function: _updateHistoryState
+* var _prefix = this.options.addrPrefix || ''
+* var table = this
+* _updateHistoryState( table,_prefix)
+* returns void
+*/
+function _updateHistoryState ( table,_prefix ) {
```

Comment:
I think we can move this function into BootstrapTable class.

Reply:
This function use _buildUrl from L42-L66

Reply:
@wenzhixin do you mean the whole extension or just this function ?
What would be improvement if we would add this to the core (if you dont mean the whole extension)

Reply:
I mean only the function _buildUrl is using by _updateHistoryState, and if you plan move function _updateHistoryState you must also move _buildUrl.